Fixes

nss_dns/dns-host.c: In function `_nss_dns_gethostbyaddr_r':
nss_dns/dns-host.c:268: error: label at end of compound statement
make[2]: *** [/home/dank/wk/crosstool-0.28-rc4h/build/i686-unknown-linux-gnu/gcc-3.4.0-20040406-glibc-2.1.3/build-glibc/resolv/dns-host.os] Error 1
make[2]: Leaving directory `/home/dank/wk/crosstool-0.28-rc4h/build/i686-unknown-linux-gnu/gcc-3.4.0-20040406-glibc-2.1.3/glibc-2.1.3/resolv'
make[1]: *** [resolv/others] Error 2

when building with gcc-3.4.0

--- glibc-2.1.3/resolv/nss_dns/dns-host.c.old	Sun Apr 11 20:32:47 2004
+++ glibc-2.1.3/resolv/nss_dns/dns-host.c	Sun Apr 11 20:33:07 2004
@@ -264,7 +264,7 @@
       strcpy (qp, "ip6.int");
       break;
     default:
-      /* Cannot happen.  */
+      ; /* Cannot happen.  */
     }
 
   host_buffer = (querybuf *) malloc (sizeof (*host_buffer));
